build-tools/api-xml-adjuster/Makefile

```diff
@@ -119,3 +122,9 @@ $(DOCS_DIR)/docs-api-24: docs-24_r01.zip
 
 $(DOCS_DIR)/docs-api-25: docs-25_web.zip	# Google hadn't published docs component yet!
 	$(call extract-docs,$<,docs)
+<<<<<<< HEAD
```

Comment:
Was there a broken merge somewhere? This looks like merge conflict output, and presumably should be removed. (As-is, this is an invalid `Makefile`).